### ISSUE TITLE
Set checked state of Table row when rendering

### ISSUE THREAD

**Issue by CarloPalinckx** [MyOnlineStore/bricks#195]
### Is your feature request related to a problem? Please describe ⚡️
Currently, the Table does not support a way to check rows from outside. You cant render a table with all rows checked for instance.

### Describe the solution you'd like ✨
I think the best solution for this would be to add an extra property to the rows prop. So instead of: 

```js
rows={[
    { id: '12345', cells: [...yourCells] }
]}
```
It could be:

```js
rows={[
    { id: '12345', checked: true, cells: [...yourCells] }
]}
```

**Comment by hborghols** [comment]
Being able to check rows from outside `Table` seems like something that may be useful in a lot of situations 👌. This solution would be fine I think.

**Comment by woutsluiter** [comment]
Looks good to me 👍

**Comment by hborghols** [comment]
Also `todo`?

**Comment by CarloPalinckx** [comment]
Closed with #212
